src/pages/permission_sets/PermissionSetPermissionItem.jsx

```diff
@@ -99,7 +99,7 @@ const PermissionSetPermissionItem = observer(() => {
         {...inputProps}
         {...l10n.permission_item.display.description}
         subcategory={l10n.categories.permission_item_display}
-        field="description"
```

Comment:
so the problem with the descriptions seems to be that I just added two description fields at the same level. all of the other things that have an internal name/description don't also have a display description at the same level. I should have put the permission item's displayed title/subtitle/description under a 'display' subkey or something.

Let's just remove the first 'internal' description field and keep the display one as-is